Sources/Web5/Dids/BearerDID.swift

```diff
@@ -45,6 +45,35 @@ public struct BearerDID {
         return did[keyPath: member]
     }
 
+    /// Returns a `BearerDIDSigner` that can be used to sign messages, credentials, or arbitrary data
+    ///
+    /// If given, the `verificationMethodID` parameter is used to select a key from the
+    /// verification methods present in the `DIDDocument`. If `verificationMethodID` is not
+    /// provided, the first verificationMethod in the `DIDDocument` will be used.
+    ///
+    /// - Parameters:
+    ///   - keyAlias: Alias of the key that will be used for
+    public func getSigner(verificationMethodID: String? = nil) throws -> BearerDIDSigner {
+        let verificationMethod: VerificationMethod?
+        
+        if let verificationMethodID {
+            verificationMethod = document.verificationMethod?.first { $0.id == verificationMethodID }
+        } else {
+            verificationMethod = document.verificationMethod?.first
```

Comment:
I'm just defaulting to the first verification method for now, but we could change this. 

If we want this to differ based on the DID Method that created this `BearerDID`, we'd need to do some additional mapping of the `did.methodName` function to it's appropriate DID method.